tests/modules/test_JMeterExecutor.py

```diff
@@ -320,10 +320,12 @@ def test_resource_files_data_sources_full_form(self):
                         'path': csv_file,
                         'loop': False,
                         'quoted': True,
+                        'stop-on-eof': False,
                     }, {
                         'path': csv_file_uni,
                         'loop': False,
                         'quoted': True,
+                        'stop-on-eof': True,
```

Comment:
Whoops, it was a post-rebase issue. `loop` is still there, so tests were working as expected.

And neither JMeter nor other executors check for extra options (yet :grey_exclamation:), so tests are not expected to fail.